settings.gradle

```diff
@@ -41,5 +42,5 @@ gradle.ext.scalafmt = [
     config: new File(rootProject.projectDir, '.scalafmt.conf')
 ]
 
-gradle.ext.akka = [version : '2.6.12']
-gradle.ext.akka_http = [version : '10.2.4']
+gradle.ext.akka = [version : '2.5.32']
```

Comment:
curious: Why does this need to be bumped if we're just upgrading go? is it to support another change that was made previously and just not reflected previously?

Reply:
Right, I actually pushed this by accident. These are required to match what we have in serverless/main.